android/app/src/main/res/layout/activity_main.xml

```diff
@@ -1,9 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.recyclerview.widget.RecyclerView
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/recyclerview"
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:clipToPadding="false"
-    android:paddingTop="8dp"
-    android:paddingBottom="8dp" />
+    android:layout_height="match_parent">
+
+    <com.google.android.material.appbar.AppBarLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:liftOnScroll="true">
+
+        <com.google.android.material.appbar.MaterialToolbar
+            android:id="@+id/topAppBar"
```

Comment:
Nit: please, use snake_case for ids.

Reply:
I'm really confused about this. The Material Design doc uses camelCase: https://material.io/develop/android/components/top-app-bars/. It's very inconsistent throughout. Is snake_case the one true way?

Reply:
Google is not very consistent in their examples. It really comes down to the personal preference, and I think that having naming of `ids` consistent with the rest of resources makes sense. For example, when you reference layouts, string resources, dimens, etc, all of them are defined in the snake case.